specifyweb/frontend/js_src/lib/localization/preferences.ts

```diff
@@ -296,6 +296,12 @@ export const preferencesText = createDictionary({
     'fr-fr': 'Apparence',
     'uk-ua': 'Зовнішній вигляд',
   },
+  buttonLight: {
```

Comment:
be consistent. if localization key says "button", than language string should say "Button". or  the opposite, if you want language string to say "Buttons", then use "buttons" in the localization key.

This way, when looking at the localization string in the code, you can easily guess what it would print